apps/meteor/app/api/server/v1/channels.ts

```diff
@@ -1044,6 +1045,57 @@ API.v1.addRoute(
 	},
 );
 
+API.v1.addRoute(
+	'channels.membersByHighestRole',
+	{ authRequired: true },
+	{
+		async get() {
+			const findResult = await findChannelByIdOrName({
+				params: this.queryParams,
+				checkedArchived: false,
+			});
+
+			if (findResult.broadcast && !(await hasPermissionAsync(this.userId, 'view-broadcast-member-list', findResult._id))) {
+				return API.v1.unauthorized();
+			}
+
+			const { offset: skip, count: limit } = await getPaginationItems(this.queryParams);
+			const { sort = {} } = await this.parseJsonQuery();
+
+			check(
+				this.queryParams,
+				Match.ObjectIncluding({
+					status: Match.Maybe([String]),
+					filter: Match.Maybe(String),
+				}),
+			);
```

Comment:
no ajv? 👀

Reply:
This one used `ajv` already, so there was no use for this `check` statement. Removed it